test/functional/Expression/AbstractLeftAssociativeExpressionTest.php

```diff
@@ -1,41 +1,42 @@
 <?php
 
-namespace Dhii\Expression\Test;
+namespace Dhii\Expression\FuncTest\Expression;
 
-use \Dhii\Expression\AbstractLeftAssocOperatorExpression;
-use \Dhii\Evaluable\EvaluableInterface;
-use \Xpmock\TestCase;
+use Dhii\Expression\Expression\AbstractLeftAssocOperatorExpression;
+use Dhii\Evaluable\EvaluableInterface;
+use Xpmock\TestCase;
 
 /**
- * Tests {@see \Dhii\Expression\AbstractLeftAssocOperatorExpression}.
+ * Tests {@see Dhii\Expression\AbstractLeftAssocOperatorExpression}.
  *
- * @since [*next-version*]
+ * @since 0.1
  */
-class AbstractLeftAssocOperatorExpressionTest extends TestCase
+class AbstractLeftAssociativeExpressionTest extends TestCase
 {
-
     /**
      * The name of the test subject.
+     *
+     * @since 0.1
      */
-    const TEST_SUBJECT_CLASSNAME = 'Dhii\\Expression\\AbstractLeftAssocOperatorExpression';
+    const TEST_SUBJECT_CLASSNAME = 'Dhii\\Expression\\Expression\\AbstractLeftAssocOperatorExpression';
 
     /**
      * Creates an instance of the test subject.
      *
-     * @since [*next-version*]
+     * @since 0.1
      *
      * @return AbstractLeftAssocOperatorExpression
      */
     public function createInstance()
     {
         $mock = $this->mock(static::TEST_SUBJECT_CLASSNAME)
-            ->_initBuffer(function() {
-                return null;
+            ->_initBuffer(function () {
```

Comment:
`return` is identical to `return null`, and is exactly what happens if you don't explicitly return anything. Also, not passing anything to mocked method will return `null` from that method, This means that you can re-write this as simply `_initBuffer()`.